packages/url_launcher/url_launcher_web/lib/src/link.dart

```diff
@@ -207,7 +207,7 @@ class LinkViewController extends PlatformViewController {
     pushRouteNameToFramework(context, routeName);
   }
 
-  Uri _uri;
+  late Uri _uri;
```

Comment:
We should accept null values here.

Reply:
Good to know! Thanks for pointing this out, I'll fix it ASAP!

Reply:
I've made this `Uri?`, and also added a small test to ensure null Uris are handled properly!